containers/claims/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+from fastapi.testclient import TestClient
+
+from ..main import app
+
+@pytest.fixture(scope="module")
+def test_app():
```

Comment:
This sounds like a test. Rename to avoid confusion

```suggestion
def client_fixture():
```

Reply:
Yep - copied these from other services mostly as-is, not 100% up on python unit test style conventions.